### PR TITLE
Error in ASPxSpreadsheet test in IE fixed

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @churkin

### CODE REVIEW

File: src/client/sandbox/event/simulator.js [hunk 1]
```diff
@@ -531,8 +531,8 @@ export default class EventSimulator {
 
             // NOTE: In IE11, iframe's window.event object is null. We need to set
             // iframe's window.event object manually by using window.event (B254199).
-            if (browserUtils.version === 11 && isElementInIframe) {
-                Object.defineProperty(domUtils.getIframeByElement(el).contentWindow, 'event', {
+            if (browserUtils.version === 11 && isElementInIframe && iframe) {
```
Comment (churkin) [inline src/client/sandbox/event/simulator.js]: `&& isElementInIframe` check is excess.
See https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/event/simulator.js#L440

Comment (churkin) [inline src/client/sandbox/event/simulator.js]: It would be ideal to write a test :)

**Comment by helen-dikareva** [comment]
@VasilyStrelyaev

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR

**Pull request merged**